core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/edit-ema-editor.component.spec.ts

```diff
@@ -383,7 +389,33 @@ const createRouting = (permissions: { canEdit: boolean; canRead: boolean }) =>
                 provide: DotPageApiService,
                 useValue: {
                     get({ language_id }) {
+                        // We use the language_id to determine the response, use this to test different behaviors
                         return {
+                            6: of({
```

Comment:
this object could be inside an objet mock in a file, to have less lines in the test.

Reply:
I moved other things from this file to centralized one, I think this object should stay here at the moment. Maybe we will change how this works